src/main.ts

```diff
@@ -48,7 +48,6 @@ async function run(): Promise<void> {
     } else {
       await createComment(octokit, repo, prNumber, commentBody)
     }
-
     core.info('Published report')
```

Comment:
🚫 **[eslint]** <i18n-text/no-en> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>English text in string literals is not allowed